benchmark/lib/experiment_server.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
```

Comment:
is it possible to add make this macro the default for every file instead of always adding it?